src/components/tags/tags.module.scss

```diff
@@ -0,0 +1,22 @@
+@import "/src/css/typography.css";
+
+.tag {
+  border: 1.5px solid rgba(155, 155, 155, 0.5);
+  border-radius: 6.49875px;
+  height: 1.6rem;
+  margin: 0 1rem 1rem 0;
+  text-align: center;
+  padding: 0 1rem;
+  line-height: 1.7rem;
+  font-family: 'gilroy';
+}
+
+.tag::first-letter {
+  text-transform: capitalize;
+}
```

Comment:
Does it make the first letter capitalized?

Reply:
Yes

Reply:
Magic!!